File: executor/src/main/java/com/codeloom/executor/repository/TestCaseRepository.java
package com.codeloom.executor.repository;

import com.codeloom.executor.model.TestCase;
import java.util.*;
import org.springframework.data.repository.CrudRepository;

public interface TestCaseRepository extends CrudRepository<TestCase, UUID> {
  List<TestCase> findByProblemId(long id);
}

File: executor/src/main/kotlin/com/codeloom/executor/repository/TestCaseRepository.kt
package com.codeloom.executor.repository

import com.codeloom.executor.model.TestCase
import org.springframework.data.repository.CrudRepository
import java.util.UUID

interface TestCaseRepository : CrudRepository<TestCase, UUID> {
    fun findByProblemId(problemId: Long): List<TestCase>
}
